packages/bits/src/lib/select-v2/base-select-v2.ts

```diff
@@ -433,4 +444,29 @@ export abstract class BaseSelectV2 implements AfterViewInit, AfterContentInit, C
             resizeObserver.unobserve(this.elRef.nativeElement);
         });
     }
+
+    /** This helps to dynamically set minHeight for overlay to avoid issues with double scroll
+     Overlay minHeight should be bigger than cdkVirtualScroll container
+     */
```

Comment:
JSDoc formatting adjustment:
```suggestion
    /**
     * This helps to dynamically set minHeight for overlay to avoid issues with double scroll
     * Overlay minHeight should be bigger than cdkVirtualScroll container
     */
```